src/cargo/util/toml/mod.rs

```diff
@@ -721,6 +726,7 @@ impl TomlManifest {
                 config: config,
                 warnings: &mut warnings,
                 platform: None,
+                features: &Features::default(), // @alex: is this right?
```

Comment:
Oh here I think the code that extracts a set of unstable features from a manifest should be mirrored when dealing with the "virtual manifests", I don't think we'll want to pass down the default here